src/System.Management.Automation/engine/PSConfiguration.cs

```diff
@@ -4,29 +4,25 @@
 using System.Threading;
 using Newtonsoft.Json;
 using Newtonsoft.Json.Linq;
+using System.Management.Automation.Internal;
 
 namespace System.Management.Automation.Configuration
 {
+    internal enum ConfigScope
+    {
+        // SystemWide configuration applies to all users.
+        SystemWide = 0,
+
+        // CurrentUser configuration applies to the current user.
+        CurrentUser = 1
```

Comment:
Previously I have put a comment that we should consider Side-By-Side local config too. It seems we should add new value here.

Reply:
The `SystemWide` option is using the config file under $PSHome, it is the system-wide config in terms of a specific PSCore deployment, but in side-by-side scenario where multiple PSCore deployments are involved, each PSCore deployment will have its own system-wide config file.

Reply:
I do not understand completely how it should work on Unix but well imagine how it should work on Windows. What about `/etc/PowerShell`? It seems it is system-wide and should works like Windows GPOs.